platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/RuntimeWorkspaceRegistry.java

```diff
@@ -272,7 +246,7 @@ public void update(RuntimeWorkspaceImpl update) throws ServerException, NotFound
      *         workspace identifier to perform check
      * @return true if workspace is running, otherwise false
      */
-    public boolean isRunning(String workspaceId) {
+    public boolean hasRuntime(String workspaceId) {
```

Comment:
> What "runtime" is in your vision?

 its rather API vision than mine, if you are interested review [related models](https://github.com/codenvy/che-core/tree/master/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/model).

> Why do you rename this method?

I did it because `isRunning` is confusing name and may be used wrong.
As you can see the name of the class is `RuntimeWorkspaceRegistry` which is responsible for managing `Runtime` representations of the `UsersWorkpace`.
In this case consider the following example:

``` 
if (workspaceRegistry.hasRuntime(workspaceId)) {
    // ...
}
```
Which actually means _If  workspace has runtime representation_. 
When: 

``` 
if (workspaceRegistry.isRunning(workspaceId)) {
    // ...
}

```
is confusing and may be considered as if _If workspace has runtime representation with  [RUNNING](https://github.com/codenvy/che-core/blob/master/platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/model/workspace/WorkspaceStatus.java#L59)  status_